sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/reporting/EvalPDFReportBuilder.java

```diff
@@ -202,6 +202,9 @@ public void addSectionHeader(String headerText,boolean lastElementIsHeader, floa
     	headerPara.setSpacingAfter(SPACING_AFTER_HEADER);
     	this.addElementWithJump(headerPara, true);
     }
+    public void addSectionHeader(String headerText,boolean lastElementIsHeader) {
+        this.addSectionHeader(headerText, lastElementIsHeader, 12.0f);
```

Comment:
Hardcoded 12pt font size (assuming theSize parameter in addSectionHeader is in pt's). Could probably be done better, open to suggestions